spec/rails_helper.rb

```diff
@@ -1,4 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'database_cleaner'
```

Comment:
- [ ] You are requesting for database cleaner, a gem, but this gem does not exist in [your Gemfile](https://github.com/Bellagirl-maker/blog_app/blob/d07ed90b8fdaeb3e629315745cfc5a86e83e8ddc/Gemfile), so I cannot run your tests due to this error. Kindly fix that.

  ![image](https://github.com/Bellagirl-maker/blog_app/assets/81468675/ddf0bf69-af34-4e6e-950a-477452c7273e)

Reply:
@julie-ify done with this change